qa-test-app/src/main/java/com/mapbox/navigation/qa_test_app/view/AlternativeRouteActivity.kt

```diff
@@ -346,6 +397,11 @@ class AlternativeRouteActivity : AppCompatActivity(), OnMapLongClickListener {
         }
     }
 
+    private val fasterRouteObserver = NewFasterRouteObserver { newFasterRoute: NewFasterRoute ->
```

Comment:
@korshaknn , @Zayankovsky , your feedback about faster routes API will be super valuable as you guys will be the first users :)